tests/tasks/setup/eks/awscli-role.yaml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
I would see this task to more generic enough like as simple as doing any role creation, doesn't have to be specific to eks service role creation (or) eks node role creation. Makes sense ? Could we abstract that out ? Basically a task which takes input to create a specific role using a path to the policies.

Reply:
Right..as discussed offline I have made this change in the new commit.